pkgs/tools/security/flare-floss/default.nix

```
@@ -2,44 +2,73 @@
 , python3
 , fetchFromGitHub
 }:
+let
+  python3' = python3.override {
```

```suggestion
  py = python3.override {
```
we normally name it like this